src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -320,6 +323,47 @@ public long StartBackupTask(long taskId, bool isFullBackup)
             return CreateBackupTask(periodicBackup, isFullBackup, SystemTime.UtcNow);
         }
 
+        public async Task Delay(long taskId, TimeSpan? delayInHrs, X509Certificate2 clientCert)
+        {
+            foreach (var periodicBackup in _periodicBackups)
+            {
+                var runningTask = periodicBackup.Value.RunningTask;
+                if (runningTask == null || runningTask.Id != taskId)
+                    continue;
+
+                var delayUntil = DateTime.UtcNow.AddTicks(delayInHrs.Value.Ticks);
+
+                var command = new DelayBackupCommand(_database.Name, RaftIdGenerator.NewId())
+                {
+                    TaskId = periodicBackup.Key,
+                    DelayUntil = delayUntil
+                };
+
+                (long index, _) = await _database.ServerStore.SendToLeaderAsync(command);
```

Comment:
what if the cluster is down? we will not be able to delay this specific node?

Reply:
Done.
Now cluster command is wrapped with try-catch and logging this information.
The test implemented proves we can still delay tasks in a cluster-down condition.